inst/doc/Data_Sources_cache/html/__packages

```diff
@@ -0,0 +1,9 @@
+base
+sp
+raster
+zoon
+maxlike
+dismo
+assertthat
+MASS
+spocc
```

Comment:
I can't find anything about in online, but I'm guessing knitr? There is one for each of the guides in their respective cache folders.